### DISCUSSION TITLE
How do I serve videos from sftp?

### DISCUSSION THREAD

**Discussion by SRoy89** [Kitware/trame#339] [General] [answered]
I found [this discussion](https://github.com/Kitware/trame/discussions/331#discussioncomment-6993343):
> If you aim to serve your own video, you will have to call `server.enable_module({ "serve": { "/videos": "/path/with/your/videos" }})`

However, the videos aren't in one single directory in my sftp server, they are in different folders. How do I do it?
Also, do I call the `server.enable_module` in `configuration.php` file somewhere?

I'm new to this, so I'm not sure of this and other unrelated things such as:
- As I have enabled sftp, that is the only thing that shows up in the home directory, nothing local shows up.
- There are also no thumbnails in list view on the files, pictures, videos, is it possible to add them?

Thanks!

**Comment by jourdain** [comment, marked as answer]
Hum, I don't understand what you are trying to do with PHP, SFTP and trame?
I guess I will let you figure that part out first.

**Comment by SRoy89** [comment]
I figured it out some time ago, thanks for replying!
